src/components/base/outline-accordion/outline-accordion.css

```diff
@@ -0,0 +1,14 @@
+.accordion-buttton {
```

Comment:
```suggestion
.accordion-button {
```

Nitpick: `buttton` should be spelled with two `t`s.

Reply:
Not nitpick at all That's me being sloppy dev. Good catch!